test/sequencer_test.cc

```diff
@@ -162,29 +165,14 @@ TEST_F(SequencerTestWithTimerEmulation, RateLimiterInteraction) {
   sequencer.waitForCompletion();
 }
 
-TEST_F(SequencerTestWithTimerEmulation, StartingLate) {
```

Comment:
This is probably related to one of my previous questions - it is not clear to me why / how was this behavior removed. Can you help me understand?